daml-lf/data/src/main/scala/com/digitalasset/daml/lf/data/FrontStack.scala

```diff
@@ -3,11 +3,10 @@
 
 package com.daml.lf.data
 
-import FrontStack.{FQ, FQCons, FQEmpty, FQPrepend}
-
-import scalaz.{Applicative, Equal, Traverse}
+import com.daml.lf.data.FrontStack.{FQ, FQCons, FQEmpty, FQPrepend}
```

Comment:
"Clean up imports" doesn't.
```suggestion
import FrontStack.{FQ, FQCons, FQEmpty, FQPrepend}
```

Reply:
Clearly. I'll fix this and the one in `BackStack`.